pkg/controller/cloneset/cloneset_controller.go

```diff
@@ -498,3 +505,27 @@ func (r *ReconcileCloneSet) truncateHistory(
 	}
 	return nil
 }
+
+func (r *ReconcileCloneSet) claimPods(instance *appsv1alpha1.CloneSet, pods []*v1.Pod) ([]*v1.Pod, error) {
```

Comment:
Could you add some tests for `claimPods`?

Reply:
done.